freq/lizard_connector.py

```diff
@@ -445,7 +445,9 @@ def ts_to_dict(self, statistic=None, values=None,
             except ValueError:
                 timestamps = [np.nan, np.nan]
             if not len(result['events']):
-                ts.append([np.nan for _ in range(len(stats1) + 2)] + timestamps)
+                y = 2 if statistic == 'difference (mean last - first year)' \
+                    else 0
+                ts.append([np.nan for _ in range(len(stats1) + y)] + timestamps)
```

Comment:
_ wordt ook vaak als niet gebruikte variabele naam gebruikt toch? Dat is hoe ik hem hier gebruik tenminste.